src/client/src/containers/ViewLicensesModal/index.tsx

```diff
@@ -0,0 +1,67 @@
+import classnames from "classnames";
+import * as React from "react";
+import { connect } from "react-redux";
+import { Dispatch } from "redux";
+import { defineMessages, InjectedIntlProps, injectIntl } from "react-intl";
+
+import { AppState } from "../../reducers";
+import buttonStyles from "../../css/buttonStyles.module.css";
+import styles from "./styles.module.css";
+import asModal from "../../components/Modal";
+import RootAction from "../../actions/ActionType";
+import { closeModalAction } from "../../actions/modalActions/modalActions";
+import Licenses from "../Licenses";
+import { isViewLicensesModalOpenSelector } from "../../selectors/modalSelector";
+import { MODAL_TYPES } from "../../actions/modalActions/typeKeys";
+
+interface IStateProps {
+  isModalOpen: boolean;
+}
+
+interface IDispatchProps {
+  closeModal: () => any;
+}
+
+type Props = IStateProps & InjectedIntlProps & IDispatchProps;
+
+const messages = defineMessages({
+  closeModalLabel: {
+    id: "viewLicensesModal.closeModalLabel",
+    defaultMessage: "Close"
+  }
+});
+
+const ViewLicensesModal = ({ intl, closeModal }: Props) => {
```

Comment:
That's a good candidate for unit tests ;)

Reply:
Will log it 👍